Makefile

```
@@ -648,14 +648,27 @@ draft-advisory-check: ## ⛔ 擋住「有未發布 draft advisory 就打 tag」
+# is not a new class of failure — just a slower step.
 # Needs docker (buildx) + trivy on PATH — run on the maintainer machine /
 # dev container, not a bare host. docker-build-all is a HARD gate (build
 # failure aborts pre-tag); trivy-scan-all is informational (prints CVEs but
 # does not block — same stance as Layer 1 / #448).
+# ⛔ The image list here is pinned against nightly-image-scan.yaml,
+# component-docker-build.yaml and the Dockerfile inventory by
+# test_selfbuilt_matrix_covers_every_dockerfile (#1337) — adding a build here
+# without adding it there (or vice versa) is a hard test failure, by design.
 .PHONY: docker-build-all trivy-scan-all
-docker-build-all: ## 建 5 個 production component image（local --load，無 push；#474 Layer 2）
+docker-build-all: ## 建 7 個 self-built image（local --load，無 push；#474 Layer 2）
```

checkmake (0.3.2)

[warning] Target body for "docker-build-all" exceeds allowed length of 5 lines (16).

(maxbodylength)